lib/hcai_form.dart

```diff
@@ -1135,8 +1135,8 @@ class _HcaiFormPageState extends State<HcaiFormPage> {
                 if (setDate['calculatedKey'] == 'dateofCautiEvent') {
                   this._values['dateOfCautiEvent'] =
                       DateFormat('MMMM d, y').format(dateToConsider);
-                  this._values['infectionWindowPeriod'] =
-                      Helper.rangeInText(this._values['dateofCautiEvent']);
+                  this._values['infectionWindowPeriod'] = Helper.rangeInText(
+                      this._values['dateofUrineSampleCollectionforCulture']);
```

Comment:
Isn't it's base on cauti event date and cauti event date is the one which is snallest among those

Reply:
yes it was but yesterday mubassir and bushra asked me to change it to urine sample date despite what ever cauti doe is